### DISCUSSION TITLE
Should jbang use the Maven local repository from ~/.m2/settings.xml?

### DISCUSSION THREAD

**Discussion by fbricon** [jbangdev/jbang#1522] [Ideas]
JBang now uses Maven's ~/.m2/settings.xml for everything auth related, but looks like custom `<localRepository>` setting is ignored. I see from calls to `jbang info tools` that all dependencies are read from `~/.m2/repository`. 
If a user set `<localRepository>`, then it'd be faster to read existing dependencies there, rather than (re)downloading them to  `~/.m2/repository`.
Asking here since it might be a design decision.

**Comment by maxandersen** [comment]
A good question. I guess if we are parsing m2e settings.xml we might as well honor localrepo too. 

My hesitance that I really don't want m2e settings.xml to be required in long run.
